pkg/api/vlabs/const.go

```diff
@@ -30,6 +30,7 @@ const (
 const (
 	Ubuntu            Distro = "ubuntu"
 	Ubuntu1804        Distro = "ubuntu-18.04"
```

Comment:
It bugs me that we have to define these consts twice...

Reply:
Yes it's called a brittle dependency hierarchy :)